awx_collection/roles/projects/README.md

```diff
@@ -0,0 +1,133 @@
+# controller_configuration.projects
+## Description
+An Ansible Role to create Projects on Ansible Controller.
+
+## Requirements
+ansible-galaxy collection install  -r tests/collections/requirements.yml to be installed
+Currently:
+  awx.awx
+  or
+  ansible.controller
```

Comment:
weird to list this as a requirement

Reply:
I lifted and shifted the entire role, all the documentation is going to need to be rewritten to remove all references like this.